src/main/python/keymaps.py

```diff
@@ -1,13 +1,15 @@
 from keycodes import Keycode
-from keymap import french, german, hungarian, norwegian, russian, swedish
+from keymap import french, german, hungarian, latam, norwegian, russian, spanish, swedish
 
 KEYMAPS = [
     ("QWERTY", dict()),
     ("French (AZERTY)", french.keymap),
     ("German (QWERTZ)", german.keymap),
     ("Hungarian (QWERTZ)", hungarian.keymap),
+    ("LatinAmerican (QWERTY)", latam.keymap),
```

Comment:
can you put "Latin American (QWERTY)" instead, with a space?

Reply:
OK.

Reply:
Done!